OpenRA.Mods.Common/Traits/CashTrickler.cs

```diff
@@ -27,6 +27,9 @@ public class CashTricklerInfo : PausableConditionalTraitInfo, IRulesetLoaded
 		[Desc("Amount of money to give each time.")]
 		public readonly int Amount = 15;
 
+		[Desc("Used for fakes, if you want to only show the cash ticks, but does not actually give the cash.")]
+		public readonly bool TransfersMoney = true;
```

Comment:
The name and description don't help me understand what this is supposed to do. TransfersMoney where? (also we don't use the term Money anywhere else outside this file). What happens to the cash that isn't actually being given?

Reply:
I'm not convinced that this flag belongs (vs having its own trait), but if it stays I would suggest renaming / redescribing in terms of being "visual only" rather than not transfering.

Reply:
changed to `TransfersCash` and made the decsription clearer